CONTRIBUTING.md

```diff
@@ -12,14 +12,7 @@ Please read and follow our [Code of Conduct](https://github.com/angular/code-of-
 ## <a name="issue"></a> Open an Issue
 
 If you find a bug in AngularHint, you can help us by submitting an issue to our
-[GitHub Repository](https://github.com/angular/angular-hint). If you can trace the issue to a specific AngularHint module, please go
-to the module's repository to submit the issue. The current repositories for AngularHint modules are:
-  - [Angular Hint Controllers](https://github.com/angular/angular-hint-controllers)
-  - [Angular Hint Directives](https://github.com/angular/angular-hint-directives)
-  - [Angular Hint DOM](https://github.com/angular/angular-hint-dom)
-  - [Angular Hint Events](https://github.com/angular/angular-hint-events)
-  - [Angular Hint Interpolation](https://github.com/angular/angular-hint-interpolation)
-  - [Angular Hint Modules](https://github.com/angular/angular-hint-modules)
```

Comment:
TBH, I have no idea what happened to them :grin:
I know the repositories do not exist any more (although they used to) and [src/modules][1] has some files that seem to correspond to some (although not all) of the modules. So I assume some got merged, some got removed(?).

[1]: https://github.com/angular/angular-hint/tree/master/src/modules